tools/travis_script.py

```diff
@@ -135,10 +135,15 @@ def build_iotjs(buildtype, args=[], env=[]):
         for buildtype in BUILDTYPES:
             if buildtype == 'release':
                 set_release_config_tizenrt()
+            # Fixme: EXTRA_LIBPATHS and EXTRA_LIB can be deleted
+            # when TizenRT uses jerry-ext
```

Comment:
Use uppercase letters for fixme, and a dot is missing from the end of the sentence.

Reply:
Thanks, i missed that. Updated the PR accordingly.